lib/Thruk/Config.pm

```diff
@@ -533,6 +533,7 @@ sub set_default_config {
         'cookie_auth_domain'                => '',
         'perf_bar_mode'                     => 'match',
         'sitepanel'                         => 'auto',
+        'site_global_switch'                => 1,
```

Comment:
i wouldn't enable it right from the start, maybe enable it when there are more than 3 backends. It doesn't make sense for a very small number of backends.